instrumentation/que/lib/opentelemetry/instrumentation/que/instrumentation.rb

```diff
@@ -60,6 +63,7 @@ def gem_version
 
         def patch
           ::Que::Job.prepend(Patches::QueJob)
+          ::Que::Poller.prepend(Patches::Poller)
```

Comment:
@open-telemetry/ruby-maintainers Should this check the `trace_poller` option here and only mixing if the option was set to `true`?

That would make it symmetric with how the top level instrumentation works, which skips the `install` block when the instrumentation `enabled` is `false`: 

https://github.com/open-telemetry/opentelemetry-ruby/blob/main/instrumentation/base/lib/opentelemetry/instrumentation/base.rb#L216

Reply:
We want to do this even if this is `false`. Because then we use `OpenTelemetry::Common::Utilities.untraced { super }` in the patched method. This makes sure the inner spans (postgres spans) are not traced.

Reply:
Thanks for the explanation. I will follow up with some questions about this inline as part of `que/patches/poller`